src/web/app/components/sessions-table/sessions-table.component.html

```diff
@@ -54,7 +54,7 @@
           <div *ngIf="sessionsTableRowModel.responseRate.length !== 0">{{ sessionsTableRowModel.responseRate }}</div>
           <tm-ajax-loading *ngIf="sessionsTableRowModel.isLoadingResponseRate"></tm-ajax-loading>
         </td>
-        <td class="actions-cell">
+        <td class="bin-header-button">
```

Comment:
We should leave this as `actions-cell` because it's under `Action(s)` column, so please revert back this change